restore/restore.go

```diff
@@ -96,18 +96,31 @@ func GetResponse(client *http.Client, req *http.Request) *http.Response {
 	return resp
 }
 
-func (c *RestoreCommand) StartRestore(cliConnection plugin.CliConnection, serviceInstanceName string, backupId string) {
+func (c *RestoreCommand) StartRestore(cliConnection plugin.CliConnection, serviceInstanceName string, backupId string, timeStamp string, guidBool bool) {
 	fmt.Println("Starting restore for ", AddColor(serviceInstanceName, cyan), "...")
 
 	if helper.GetAccessToken(helper.ReadConfigJsonFile()) == "" {
 		errors.NoAccessTokenError("Access Token")
 	}
 
 	client := GetHttpClient()
-
-	var jsonprep string = `{"backup_guid": "` + backupId + `"}`
-	var jsonStr = []byte(jsonprep)
-	var req_body = bytes.NewBuffer(jsonStr)
+	var req_body = bytes.NewBuffer([]byte(""))
+	if guidBool == true {
+		var jsonprep string = `{"backup_guid": "` + backupId + `"}`
+		var jsonStr = []byte(jsonprep)
+		req_body = bytes.NewBuffer(jsonStr)
+	} else {
+		time, err := time.Parse(time.RFC3339, timeStamp)
+		if err != nil {
+			fmt.Println(AddColor("FAILED", red))
+			fmt.Println(err)
+			fmt.Println("Please enter time in ISO8061 format, example - 2018-11-12T11:45:26.371Z")
```

Comment:
If you support additional formats, can you give an example of them here too?
May be one or two more? Just to help the user understand his options.
e.g. `2018-11-12T11:45`

Reply:
Format should be ISO8601 .